CHANGELOG/v1.6.1.md

```diff
@@ -0,0 +1,47 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.29.x
+- Workload Cluster: v1.23.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+
+* Kubernetes v1.29 is now supported
+
+## Changes since v1.6.0
+## :chart_with_upwards_trend: Overview
+- 30 new commits merged
+- 2 feature additions ✨
```

Comment:
are we missing the `## :sparkles: New Features` section for this one?

Reply:
There weren't any new features. There were two that were tagged as features but were either a bug fix or test failure.

Reply:
per release call- added 1 to make consistent with 1.5.4